DQM/L1TMonitor/src/L1TStage2uGMT.cc

```diff
@@ -284,6 +351,30 @@ void L1TStage2uGMT::bookHistograms(DQMStore::IBooker& ibooker, const edm::Run&,
   ugmtMuonPhiAtVtxEmtf = ibooker.book1D("ugmtMuonPhiAtVtxEmtf", "uGMT Muon #phi at vertex for EMTF Inputs", 126, -3.15, 3.15);
   ugmtMuonPhiAtVtxEmtf->setAxisTitle("#phi at vertex", 1);
 
+  ugmtMuonDEtavsPtBmtf = ibooker.book2D("ugmtMuonDEtavsPtBmtf", "uGMT Muon from BMTF #eta_{at vertex} - #eta_{at muon system} vs p_{T}", 80, 0, 80, 50, -0.2, 0.2);
```

Comment:
That would leave the histogram with little value for the DQM. But I can probably get a factor 2.5-3 less bins without sacrificing too much information.